pkg/conduit/config.go

```diff
@@ -219,6 +226,19 @@ func (c Config) validateSchemaRegistryConfig() error {
 		if c.SchemaRegistry.Confluent.ConnectionString == "" {
 			return requiredConfigFieldErr("schema-registry.confluent.connection-string")
 		}
+		if c.SchemaRegistry.Confluent.Authentication.Type == "basic" {
```

Comment:
Nitpick: it would be good to fail if the type is something other than basic or bearer.